wallet/types/evm_types.go

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package types
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/core/types"
+)
+
```

Comment:
This file contains the input that can be fed into our build, sign, send Tx operations similar to how we do it for Avalanche Txs